content/en/docs/setup/upgrade/gateways/index.md

```diff
@@ -1,19 +1,20 @@
 ---
-title: Configuring and managing gateways with multiple revisions
-description: Configuring and managing gateways with multiple revisions.
-weight: 25
-keywords: [kubernetes,upgrading]
+title: Managing Gateways with Multiple Revisions
```

Comment:
Per https://github.com/istio/istio.io/pull/8299#issuecomment-717550680
let' mark this page experimental for now.